### ISSUE TITLE
When upgrading the Lambo config, the comment setting the time in the old config doesn't respect my time zone

### ISSUE THREAD

**Issue by mattstauffer** [tighten/lambo#146]
Instead it uses UTC.

![image](https://user-images.githubusercontent.com/151829/106934388-0fe45480-66e8-11eb-92d9-853d85816a40.png)

**Comment by jonsugar** [comment]
@mattstauffer This seems to be a PHP and/or Laravel Zero issue. 

In `vendor/laravel-zero/foundation/src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php` It does the following:

```php
date_default_timezone_set($config->get('app.timezone', 'UTC'));
```

If I set `date.timezone = 'America/New_York` in php.ini it is ignored as the above implies.

If I make the following changes to  ` vendor/laravel-zero/foundation/src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php` then the php.ini value is honored.

```php
date_default_timezone_set($config->get('app.timezone', date_default_timezone_get()));
```

If I set `timezone` in `config/app.php` to `America/New_York` it is honored.

**TL;DR**
I don't think this is solvable in a nice way. In my case, I don't really notice it as I live in UTC/GMT which PHP and Laravel Zero default to. 

It seems weird to advise users to set `timezone` in `config/app.php`. I'm not sure it's correct for Laravel Zero to set the default timezone to something other than UTC.

**Comment by mattstauffer** [comment]
Hm, I sort of think it might be ideal to set it to the current user's computer's time zone instead of UTC?

**Comment by jonsugar** [comment]
PHP doesn't let you do that. You have to manually set it in php.ini or at runtime. That's the issue.

**Comment by mattstauffer** [comment]
Womp. OK.

**Comment by jonsugar** [comment]
I got annoyed by the Lambo time zone thing 🤷🏻 

After many hours of googling, and trying various fixes, apparently, PHP does have a solution. The following addition to `config/app.php` should sort it:

```php
<?php
return [
    ⁝
    'timezone' => IntlTimeZone::createDefault()->getID(),
    ⁝
];
```

the only caveat is that **ext-intl** extension must be installed but I feel like most people have that?

The above solution works on all operating systems but if ext-intl is not installed then the following works on Mac OS:

```php
<?php
return [
    ⁝
    'timezone' => exec('/bin/ls -l /etc/localtime|/usr/bin/cut -d"/" -f8-'),
    ⁝
];
```

and the following for Debian/Ubuntu distros:

```php
<?php
return [
    ⁝
    'timezone' => exec('cat /etc/timezone'),
    ⁝
];
```
